scripts/generate-apt-packages-list.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -e
+
+# This script generates a list of packages with their versions and other details to make it easier for parsing by check-repository-consistency.js
+# Outputs scripts/apt-packages-list-<arch>.txt for each architecture
+#
+# Format of each line:
+# <package_name> <repo-name> <version> <may_have_staticsplit>
+
+TERMUX_PACKAGES_DIR="$(realpath "$(dirname "$(realpath "$0")")/..")"
+repo_paths=$(jq --raw-output 'del(.pkg_format) | keys | .[]' "$TERMUX_PACKAGES_DIR/repo.json")
+
+for arch in "aarch64" "arm" "i686" "x86_64"; do
+	for repo_path in ${repo_paths[@]}; do
+		repo_name=$(jq --raw-output ".\"$repo_path\".name" "$TERMUX_PACKAGES_DIR/repo.json")
+		package_paths=($TERMUX_PACKAGES_DIR/$repo_path/*)
+		for pkg_path in "${package_paths[@]}"; do
+			(
+				. "$pkg_path/build.sh" &> /dev/null || :
+				# We are erraneously adding a revision for packages with version containing a dash when generating the apt packages in our build scripts.
+				# So reproduce the same bug here
+        # Source: scripts/build/termux_extract_dep_info.sh
```

Comment:
Use tabs for indentation here.
```suggestion
				# Source: scripts/build/termux_extract_dep_info.sh
```